app/serializers.py

```diff
@@ -1,7 +1,28 @@
 from rest_framework import serializers
-from .models import Poll
+from .models import Poll, Question, User
+
+class QuestionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Question
+        fields = ['title', 'choices']
 
 class PollSerializer(serializers.ModelSerializer):
+    questions = QuestionSerializer(many=True)
+    creater = serializers.EmailField(write_only=True)
+
     class Meta:
         model = Poll
-        fields = ['id', 'title', 'creater', 'questions', 'is_open']
\ No newline at end of file
+        fields = ['id', 'title', 'creater', 'questions', 'is_open']
+
+    def create(self, validated_data):
+        questions_data = validated_data.pop('questions')
+        creater_email = validated_data.pop('creater')
+
+        user, created = User.objects.get_or_create(email=creater_email)
+        
+        poll = Poll.objects.create(creater=user, **validated_data)
+        
+        for question_data in questions_data:
+            Question.objects.create(**question_data)
+        
```

Comment:
You are creating Poll &Questions but are not linking it..you also need to add questions to Poll